gitpod-io/website
# Rename support patch into config patch in the documentation

## Description
We originally called the feature that allows you to upload a file that then gets applied to the installer config `support-patch`. The idea was that Gitpod support will provide this for you. However, we have since internally only called it config patch, because that better explains what it is - a patch for the installer config. Further, the KOTS installation UI also calls it config patch.

This PR changes the documentation in roder to call it config patch:
- Reworded page that explains this (troubleshooting/config-patches) and how to apply it
- Changed all the links within the documentation
- Added a re-direct for any external links that still link to /suppport-patches

## How to test
- Go to netlify preview and navigate to docs>self-hosted>troubleshooting>config-patches
- Ensure redirect works by going to docs/self-hosted/latest/troubleshooting/support-patches

## Release Notes

```release-note
NONE
```

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/2553"><img src="https://gitpod-staging.com/button/open-in-gitpod.svg"/></a>



<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/2553"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
